core/spec/mailers/shipment_mailer_spec.rb

```diff
@@ -1,20 +1,22 @@
 require 'spec_helper'
 require 'email_spec'
 
-describe Spree::ShipmentMailer, :type => :mailer do
+describe Spree::ShipmentMailer, type: :mailer do
   include EmailSpec::Helpers
   include EmailSpec::Matchers
 
   before { create(:store) }
 
+  let(:order) { stub_model(Spree::Order, number: 'R12345') }
+  let(:shipping_method) { stub_model(Spree::ShippingMethod, name: 'USPS') }
+  let(:product) { stub_model(Spree::Product, name: %Q{The "BEST" product}, sku: 'SKU0001') }
```

Comment:
Use `%` instead of `%Q`.<br>Do not use `%Q` unless interpolation is needed.  Use `%q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.